content/03.text-mining.md

```diff
@@ -7,8 +7,51 @@
 
 ### Constructing Databases and Manual Curation
 
-1. Talk about papers that construct knowledge graphs without text mining approaches
-2. Discuss the positives and negatives for these methods
+Database Construction is an extensive process that can date back all the way to 1956 where the first database contained a protein sequence of the insulin molecule [@doi:10.4172/jcsb.1000081].
+This process involves gathering relevant text such as journal articles, abstracts, or web-based text.
+At this point curators can directly scan gathered text and detect relationship asserting sentences sentences (i.e. relationship extraction).
+An alternative is to take a semi-automatic approach that involves using text mining systems to speed up the curation process.
+We discuss the pros and cons of using manual curation to do relationship extraction and mention databases that extensively use this method to populate their fields.
+
+Notable databases have been constructed using an extensive amount of manual curation (Table {@tbl:manual-curated-databases}).
+For example, COSMIC [@doi:10.1093/nar/gkw1121] was constructed via a group of domain experts scanning the literature for key cancer related genes.
+This database has reached close to 45k entries in 2016 and continues to grow throughout the years [@doi:10.1093/nar/gkw1121].
+Studies have shown that these databases contain relatively precise data, but in low quantities [@doi:10.1038/nmeth1209-860; @doi:10.1038/nmeth.1284; @doi:10.1093/database/bau058; @doi:10.1093/nar/gku1205; @doi:10.1186/s12859-018-2103-8; @doi:10.1093/database/bax043].
+This happens because the high publication rate is too much for curators to keep up [@doi:10.1007/s11192-010-0202-z].
+Plus, this process isn't completely error free as databases were found to be missing annotations and entries [@doi:10.1093/bioinformatics/btm229].
+These combined findings highlight a significant need for approaches that  can scale up to an increasing publication rate and improve overall recall of relevant sentences.
+
+There have been attempts to speed up the manual curation process via semi-automatic ways [@doi:10.1016/j.jbi.2010.11.001; @doi:10.1093/database/bau067; @doi:10.1093/database/bas010; @doi:10.1145/3269206.3269229; @doi:10.1093/database/baz068; @doi:10.1186/s12859-018-2103-8; @doi:10.1186/s12859-018-2021-9].
+First step in this context is to use an automatic system to initally extract sentences from text.
+This process filters out irrelevant sentences, which means less text for  curators to sift through.
+After the pre-filtering step curators can approve or remove the identified sentences.
+This semi-automatic process was found to speed up the curation process compared to manual approach [@doi:10.1016/j.jbi.2010.11.001; @doi:10.1136/amiajnl-2013-001837].
+Despite the speed up, this process is prone to erroneous assumptions and produce bias results.
```

Comment:
The grammar here needs to be fixed. I'm also not sure what the erroneous assumptions are based on the context around this.